src/interaction/NTEInteraction.py

```
@@ -17,34 +12,13 @@ def __init__(self, *args, **kwargs):
-            btn_up = win32con.WM_MBUTTONUP
-        else:
-            btn_down = win32con.WM_RBUTTONDOWN
-            btn_mk = win32con.MK_RBUTTON
-            btn_up = win32con.WM_RBUTTONUP
-        self.post(btn_down, btn_mk, click_pos)
-        time.sleep(down_time)
-        self.post(btn_up, 0, click_pos)
-        if x >= 0 and move_Cursor:
-            time.sleep(0.1)
-            SetCursorPos(self.cursor_position)
+        # 使用 PostMessageInteraction 的后台点击逻辑，避免真实鼠标移动
```

Ruff (0.15.11)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)